p2p/exchange_test.go

```diff
@@ -18,9 +18,8 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/celestiaorg/go-header"
-	headerMock "github.com/celestiaorg/go-header/mocks"
+	headerMock "github.com/celestiaorg/go-header/headertest"
```

Comment:
Is this necessary? I think removing alias will make code more coherent.

Reply:
Agree

Reply:
Fixed